src/Core/Engine/Components/Light.hpp

```
@@ -18,91 +18,58 @@ enum class LightType
-    glm::vec3 diffuse = glm::vec3(1.0f, 1.0f, 1.0f) ;  float _pad4 = 0.0f;
-    glm::vec3 specular = glm::vec3(.05f, .05f, .05f); float _pad5 = 0.0f;
+    glm::vec3 ambient = glm::vec3(1.0f, 1.0f, 1.0f);
+    glm::vec3 diffuse = glm::vec3(1.0f, 1.0f, 1.0f) ;
+    glm::vec3 specular = glm::vec3(.05f, .05f, .05f);
+
+    PointLight() = default;
 };
- // TODO: Fix light
-struct alignas(16) SpotLightGPU : LightBaseGPU
+
+class SpotLight
```

Clang (14.0.6)

[warning] constructor does not initialize these fields: cosCutOff, cosOuterCutOff

(cppcoreguidelines-pro-type-member-init)